elements/pfe-accordion/src/pfe-accordion.json

```diff
@@ -29,19 +29,7 @@
     "attributes": {
       "title": "Attributes",
       "type": "object",
-      "properties": {
-        "on": {
-          "title": "Context",
-          "type": "string",
-          "enum": [
-            "light",
-            "dark"
-          ],
-          "default": "light",
-          "prefixed": false
-        }
-      },
-      "required": ["on"]
+      "properties": {}
```

Comment:
should these props be removed?

Reply:
Yes, no one should be manually adding the `on` attribute to a component anymore.